### DISCUSSION TITLE
How to check the size of a Collection ?

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#26954] [Q&A and General discussion]
Hi Team,

Like in MongoDB we can use collection.totalSize() to get the total size of collection. Can someone suggest me how can we check the total size of a collection in Milvus DB using python sdk or something else.

**Comment by xiaofan-luan** [comment]
That is actually a good feature.
Just to clarify size means memory usage or row count?

**Reply by navneetrajput06** [reply]
memory usage and row count both please.

**Reply by yhmo** [reply]
To get the row count of a collection in v2.3:  https://milvus.io/docs/query.md#Count-entities
For memory usage, we have a tool to estimate(but is not accurate): https://milvus.io/tools/sizing

**Reply by navneetrajput06** [reply]
How can we check the disk size for the collection in bucket. i mean the total size for a collection?

**Reply by xiaofan-luan** [reply]
you need to use s3 to check that
